services/src/main/java/org/exoplatform/officeonline/AbstractOfficeOnlineService.java

```diff
@@ -282,21 +323,26 @@ protected User getUser(String username) {
   /**
    * Explorer uri.
    *
-   * @param schema the schema
-   * @param host the host
-   * @param port the port
+   * @param platformUrl the platform url
    * @param ecmsLink the ecms link
    * @return the uri
    */
-  protected URI explorerUri(String schema, String host, int port, String ecmsLink) {
+  protected URI explorerUri(String platformUrl, String ecmsLink) {
```

Comment:
Let's make it transparent to product name (eXo Platform), instead of `platformUrl` it can be `baseUrl`.

Reply:
Fixed: https://github.com/exo-addons/office-online/pull/15/commits/0ea08afcf3872d8b6b41acb5a53931aaea00280b